RELEASE.md

```diff
@@ -0,0 +1,88 @@
+# Release 0.2.0
+
+# Performance Optimization
+
+## GPU stitch fusion
+
+Make use of GPU shared memory to fuse reduce operator with its consumers into one kernel.
+It helps to accommodate complex memory-intensive computations (e.g., LayerNorm, SoftMax) into one kernel,
+reducing off-chip memory traffics and overhead of kernel scheduling and launching.
+It implements partial functions described in paper [AStitch](https://dl.acm.org/doi/abs/10.1145/3503222.3507723).
+It is currently under refactoring to enhance the robustness, for which it is not enabled by default.
+Users of BladeDISC can enable it by setting the environment variable `DISC_ENABLE_STITCH=true`.
+
+## GEMM merging
+
+Support two types of GEMM merging optimization.
+One is to merge two GEMMs sharing the same operand into a single GEMM.
+The other one is to merge two GEMMs with the same shape into a batched GEMM.
+The GEMM merging optimization helps to increase hardware utilization and to reduce kernel launch overhead.
+
+## CPU GEMM/Convolution weight pre-packing optimization
+
+Support weight pre-packing optimization for convolution (calling mkldnn library) and GEMM (calling mkldnn/onednn/acl libraries) operations.
+
+## Convolution layout optimization and transpose elimination
+
+Support to transform the layout of convolution operator to the friendliest format on the specific device (i.e., either CPU or GPU).
+Most of the introduced transpose operators can be eliminated in a following transpose-simplifier pass.
+
+## Other optimizations
+* Optimize the schedule selection strategy for reduce operator on GPU to enhance thread-level-parallelism.
+* Algebraic simplification for operators like power.
+* Support to fuse splat constant operator with its consumers, reducing memory access overhead.
+Refer to [issue](https://github.com/alibaba/BladeDISC/issues/113).
+
+# Function Enhancement
+
+## CPU end-to-end optimization
+
+Support end-to-end optimization for X86 and AArch64 CPUs.
+
+## TorchBlade/TensorFlowBlade clustering and optimizing with TensorRT
+
+According to the supported operators of TensorRT, cluster sub-graphs and apply TensorRT optimization for both TensorFlow and PyTorch models.
+
+## Accelerating PyTorch Training
```

Comment:
Shall we announce this? @Yancey1989 @fortianyou

Reply:
After an offline discussion, we decide to keep it as it is.